oauth-common/src/main/java/io/strimzi/kafka/oauth/common/Config.java

```diff
@@ -16,6 +16,8 @@ public class Config {
 
     public static final String OAUTH_CLIENT_ID = "oauth.client.id";
     public static final String OAUTH_CLIENT_SECRET = "oauth.client.secret";
+    public static final String OAUTH_USERNAME = "oauth.grant.password.username";
+    public static final String OAUTH_PASSWORD = "oauth.grant.password.password";
```

Comment:
I struggled on that one ;)
I wanted a property scoped to a particular grant type. and it happens that to configure the _password grant_ you need to specify a `password` property...
I wanted to allow future configuration properties such as `oauth.grant.<grant type>.<property>`
another option would have been to use `ropc` instead of `password`. 
I am going to implement your proposition.